holoviews/tests/plotting/plotly/test_callbacks.py

```diff
@@ -335,7 +335,7 @@ def testRangeCallbacks(self):
                 xyevents[4], xevents[4], yevents[4]
         ):
             assert len(xyevent) == 0
-            assert len(yevent) == 0
+            assert len(xevent) == 0
```

Comment:
Ah! That was a bug!

Reply:
Yes. Found it because it wanted to use underscore on `_xevents`.